accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -1480,17 +1517,12 @@ variable `X`, it is statically decidable which of the first two clauses apply in
 the case that `v` is `null`.
 
 **In strong checking mode**: if `e` evaluates to a value `v` and `v` has runtime
-type `S`, an instance check `e is T` occurring in a **legacy library** or an
-**opted-in library** is evaluated as follows:
-  - If `v` is `null` and `T` is a legacy type, return `LEGACY_SUBTYPE(T, NULL)
-    || LEGACY_SUBTYPE(Object, T)`
-  - Otherwise return `NNBD_SUBTYPE(S, T)`
-
-Note that in a program with no opted out libraries, the first clause can never
-apply.
+type `S`, an instance check `e is T` occurring in an **opted-in library** is
+evaluated as follows:
+  - Return `NNBD_SUBTYPE(S, T)`
 
-Note also that except in the case that `T` is of the form `X` or `X*` for some
-type variable `X`, it is statically decidable which clause applies.
+Note that it is an error to run a program containing opted-in libraries in
```

Comment:
That would be `containing legacy libraries`.

Reply:
Yes it would, wouldn't it... :) Fixed thanks!